app/jobs/add_live_result_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddLiveResultJob < ApplicationJob
+  self.queue_adapter = :shoryuken if Rails.env.production? && !EnvConfig.WCA_LIVE_SITE?
+  queue_as EnvConfig.LIVE_QUEUE if Rails.env.production? && !EnvConfig.WCA_LIVE_SITE?
```

Comment:
Can you move the `if` check into a globally accessible `WcaLive.enabled?` variable like we did with paypal?